include/functions.h

```diff
@@ -770,14 +770,14 @@ DebugDispObject* DebugDisplay_AddObject(f32 posX, f32 posY, f32 posZ, s16 rotX,
                                         f32 scaleY, f32 scaleZ, u8 red, u8 green, u8 blue, u8 alpha, s16 type,
                                         GraphicsContext* gfxCtx);
 void DebugDisplay_DrawObjects(PlayState* play);
-void func_8006450C(PlayState* play, CutsceneContext* csCtx);
-void func_80064520(PlayState* play, CutsceneContext* csCtx);
-void func_80064534(PlayState* play, CutsceneContext* csCtx);
-void func_80064558(PlayState* play, CutsceneContext* csCtx);
-void func_800645A0(PlayState* play, CutsceneContext* csCtx);
+void Cutscene_InitContext(PlayState* play, CutsceneContext* csCtx);
+void Cutscene_StartManual(PlayState* play, CutsceneContext* csCtx);
+void Cutscene_StopManual(PlayState* play, CutsceneContext* csCtx);
+void Cutscene_UpdateManual(PlayState* play, CutsceneContext* csCtx);
```

Comment:
I'm not really a fan of "manual", I would prefer something like "unscripted" but it's just a matter of personal taste so can ignore

Reply:
thats fair
I think manual better captures the fact that you quite literally have do everything. all the system will do is hide the hud and put a letterbox.
Unscripted is not wrong, but it emphasizes that less I feel. And I always naturally refer to mortia's cutscenes as "doing them manually"

any more opinions welcome

Reply:
I'm fine with manual
I don't like "unscripted" solely because it's too similar to "scripted", though I agree it's technically a better description

synonym game: I would find "homemade" funny
*Freshly baked cutscenes available at Morita&Family*